packages/onboarding/src/step-container-v2/wireframes/TwoColumnLayout/TwoColumnLayout.tsx

```diff
@@ -37,20 +37,17 @@ export const TwoColumnLayout = ( props: TwoColumnLayoutProps ) => {
 					childElements = childElements.props.children;
 				}
 
-				return Children.map( childElements, ( child, index ) => {
-					if ( ! isValidElement( child ) ) {
-						return null;
-					}
-
-					return (
+				return Children.toArray( childElements )
+					.filter( isValidElement )
```

Comment:
Let's assume this array: `[ validChild, null, validChild ]`. 

If we don't filter it before the map, the invalid child in the middle of the array will be counted in the `index`, and the third child would render with index 2, but it should actually be 1.